pymeasure/instruments/resources.py

```diff
@@ -58,3 +61,24 @@ def list_resources():
             print(e)
     rm.close()
     return instrs
+
+
+def find_serial_port(vendor_id: Optional[int] = None, product_id: Optional[int] = None,
+                     serial_number: Optional[str] = None) -> str:
```

Comment:
From old discussion, I think we are not using type hinting in pymeasure, but @bilderbuchi may know more about this.

Reply:
I stripped the hints.